test/test_helper.exs

```diff
@@ -14,13 +14,17 @@ defmodule BridgeEx.TestHelper do
     end)
   end
 
-  def reload_app do
+  def reload_app(start_prima_auth0_ex? \\ false) do
```

Comment:
```suggestion
  def reload_app(start_prima_auth0_ex?) do
```

Reply:
Sorry I am a bit noob but if do this wouldn't all usage of `reload_app` break? Or should I make everywhere explicit whether I want to `start_prima_auth0_ex?` or not (i.e. put `false`)?

Reply:
@squaini bump